website/client/js/controllers/chatCtrl.js

```diff
@@ -114,12 +114,19 @@ habitrpg.controller('ChatCtrl', ['$scope', 'Groups', 'Chat', 'User', '$http', 'A
       });
     };
 
+    function handleGroupResponse (response) {
+      $scope.group = response;
+      if (!$scope.group._id) $scope.group = response.data.data;
+    };
+
     $scope.sync = function(group) {
-      //@TODO: We need to use chat service here
-      Groups.Group.get(group._id)
-        .then(function (response) {
-          $scope.group = response.data.data;
-        })
+      if (group.name === Groups.TAVERN_NAME) {
```

Comment:
Why are we testing against the name instead of the id constant that @paglias created?

https://github.com/HabitRPG/habitrpg/blob/api-v3/website/client/js/app.js#L29